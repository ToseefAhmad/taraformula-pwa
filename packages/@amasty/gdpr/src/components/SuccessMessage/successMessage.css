.root {
  margin-bottom: 0.625rem;
  padding: 0.625rem;
  background-color: #fffbbb;
}
