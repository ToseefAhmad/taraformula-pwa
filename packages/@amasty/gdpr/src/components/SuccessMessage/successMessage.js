import React from 'react';
import { string } from 'prop-types';
import { mergeClasses } from '@magento/venia-ui/lib/classify';
import defaultClasses from '../SuccessMessage/successMessage.css';

const SuccessMessage = props => {
  const classes = mergeClasses(defaultClasses, props.classes);
  const { message } = props;

  if (!message) {
    return null;
  }

  return <p className={classes.root}>{message}</p>;
};

SuccessMessage.propTypes = {
  message: string
};

export default SuccessMessage;
