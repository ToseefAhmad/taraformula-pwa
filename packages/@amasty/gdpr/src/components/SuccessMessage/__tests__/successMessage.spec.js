import React from 'react';
import { createTestInstance } from '@magento/peregrine';
import SuccessMessage from '../successMessage';

test('renders default successMessage', () => {
  const tree = createTestInstance(<SuccessMessage message={'test'} />);
  expect(tree.toJSON()).toMatchSnapshot();
});

test('renders no message', () => {
  const tree = createTestInstance(<SuccessMessage message={''} />);
  expect(tree.toJSON()).toMatchSnapshot();
});
