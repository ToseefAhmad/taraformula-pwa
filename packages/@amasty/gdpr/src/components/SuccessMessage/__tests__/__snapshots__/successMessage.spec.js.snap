// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders default successMessage 1`] = `
<p>
  test
</p>
`;

exports[`renders no message 1`] = `null`;
